sample/README_mysql.md

```diff
@@ -0,0 +1,143 @@
+# Mysql Sample
+
+This sample is for experimenting with the Airbyte Module (ABM) mysql connetor.
+
+The server is configured to read and write a mysql table.
+
+It uses Kuberentes for the mysql chart deployment only while the Airbyte Module (ABM) runs locally without Kubernetes.
+
+## Before you begin
+
+Ensure that you have the following:
+
+- Helm 3.3 or greater must be installed and configured on your machine.
+- Kubectl 1.20 or newer must be installed on your machine.
+- Access to a Kubernetes cluster such as Kind as a cluster administrator. Kubernetes version support range is 1.24-1.22 although older versions may work well.
+- yq tool
+
+The tools can also be found in `hack/tools/bin` directory after executing `make install-tools` in the root folder.
+
+## Steps
+### Read example
+
+1. Setup and initialize mysql for reading a dataset
+
+
+    1. Create a new namespace for mysql depolyment, and set it as default:
+      ```bash
+      kubectl create namespace fybrik-airbyte-sample
+      kubectl config set-context --current --namespace=fybrik-airbyte-sample
+      ```
+
+    2. Deploy [mysql](https://bitnami.com/stack/mysql/helm) helm chart in `fybrik-airbyte-sample` namespace.
+      ```bash
+      helm repo add bitnami https://charts.bitnami.com/bitnami
+      helm install mysql bitnami/mysql -n fybrik-airbyte-sample
+      kubectl wait pod --for=condition=ready mysql-0 --namespace fybrik-airbyte-sample --timeout 20m
+      ```
+    3. Use the instructions from the helm chart notes to run a pod that is use as a client and connect to the service:
```

Comment:
"run a pod that is **used** as a client"

Reply:
Fixed. Thanks